src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.IntegrationAccount.tests.ps1

```diff
@@ -187,7 +214,215 @@ InModuleScope Arcus.Scripting.IntegrationAccount {
+                    Set-AzIntegrationAccountMaps -ResourceGroupName $resourceGroupName -Name $integrationAccountName -MapsFolder $mapsFolder
+
+                    # Assert
+                    foreach ($map in Get-ChildItem($mapsFolder) -File) {
+                        $expectedMapName = $map.Name
+                        
+                        $actual = Get-AzIntegrationAccountMap -ResourceGroupName $resourceGroupName -Name $integrationAccountName -MapName $expectedMapName
+                        $actual | Should -Not -BeNullOrEmpty
+                        $actual.CreatedTime.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ss") | Should -BeIn ($actual.ChangedTime.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ss"), $actual.ChangedTime.ToUniversalTime().AddSeconds(-1).ToString("yyyy-MM-ddTHH:mm:ss"))
+                        #$actual.CreatedTime.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ss") | Should -Be $actual.ChangedTime.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ss")
+                        $actual.CreatedTime | Should -BeGreaterOrEqual $executionDateTime
+                    }
+
+                } finally {
+                    foreach ($map in Get-ChildItem($mapsFolder) -File) {
+                        $expectedMapName = $map.Name
+                        
+                        Remove-AzIntegrationAccountMap -ResourceGroupName $resourceGroupName -Name $integrationAccountName -MapName $expectedMapName -Force
+                    }
+                }
+            }
+            It "Create multiple maps, without extension, located in a folder in an Integration Account" {
+                # Arrange
+                $resourceGroupName = $config.Arcus.ResourceGroupName
+                $integrationAccountName = $config.Arcus.IntegrationAccount.Name
+				$mapsFolder = "$PSScriptRoot\Files\IntegrationAccount\Maps"
+                $executionDateTime = (Get-Date).ToUniversalTime()
+
+                try {
+                    # Act
+                    Set-AzIntegrationAccountMaps -ResourceGroupName $resourceGroupName -Name $integrationAccountName -MapsFolder $mapsFolder -RemoveFileExtensions
+
+                    # Assert
+                    foreach ($map in Get-ChildItem($mapsFolder) -File) {
+                        $expectedMapName = $map.BaseName
+                        
+                        $actual = Get-AzIntegrationAccountMap -ResourceGroupName $resourceGroupName -Name $integrationAccountName -MapName $expectedMapName
+                        $actual | Should -Not -BeNullOrEmpty
+                        $actual.CreatedTime.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ss") | Should -BeIn ($actual.ChangedTime.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ss"), $actual.ChangedTime.ToUniversalTime().AddSeconds(-1).ToString("yyyy-MM-ddTHH:mm:ss"))
+                        #$actual.CreatedTime.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ss") | Should -Be $actual.ChangedTime.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ss")
```

Comment:
```suggestion
```

Am I repeating myself? 😄

Reply:
My bad 😄 
The worst thing about this is, while I was turning those lines in comments, I was thinking I shouldn't forget to actually remove them...